src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/dialogs/rowconfig.html

```diff
@@ -8,7 +8,12 @@
       <div class="alert alert-warn ng-scope" ng-show="nameChanged">
           Modifying a row configuration name will result in loss of
           data for any existing content that is based on this configuration.
+          Modifying only the label will not result in data loss.
```

Comment:
Suggestion @clausjensen is to perhaps to wrap this in strong tags or something to make this important note to stand out from the other part perhaps?!